odoo/addons/base/data/neutralize.sql

```diff
@@ -0,0 +1,17 @@
+-- deactivate mail servers
+UPDATE ir_mail_server
+    SET active = false;
+
+-- deactivate crons
+UPDATE ir_cron
+    SET active = false
+WHERE id NOT IN (
+    SELECT res_id FROM ir_model_data
+    WHERE model = 'ir.cron' AND name = 'autovacuum_job'
```

Comment:
Shouldn't we keep the ping cron active?

```suggestion
    SELECT res_id
      FROM ir_model_data
     WHERE model = 'ir.cron'
       AND (module, name) IN (('base', 'autovacuum_job'), ('mail', 'ir_cron_module_update_notification'))
```

Corollary question: Should the dbuuid be (optionally) changed?